common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -32,6 +32,11 @@ public class CommonValue {
     public static final String ML_TASK_INDEX = ".plugins-ml-task";
     public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 3;
     public static final Integer ML_TASK_INDEX_SCHEMA_VERSION = 1;
+
+    public static final String ML_MODEL_RELOAD_INDEX = ".plugins-ml-model-reload";
+    public static final String NODE_ID_FIELD = "node_id";
+    public static final String MODEL_LOAD_RETRY_TIMES_FIELD = "retry_times";
+    public static final Integer ML_MODEL_RELOAD_MAX_RETRY_TIMES = 2;
```

Comment:
Do we need this constant as we have setting `ML_MODEL_RELOAD_MAX_RETRY_TIMES`?

Reply:
yes, u r right, We don't need it in class `CommonValue`. But we need it in `opensearch.yml`,so we can let user define the value by himself.